src/python_testing/TC_RVCCLEANM_2_1.py

```diff
@@ -56,6 +57,10 @@ async def send_run_change_to_mode_cmd(self, newMode) -> Clusters.Objects.RvcRunM
     def write_to_app_pipe(self, command):
```

Comment:
given how many copies there are of this, it seems like it would make sense to move it to common location. Especially if you want to add a response from the pipe in the future / tune the sleep.

Can you open an issue for having the named pipe giving back an ack and add this as a todo here?